build/IPDK_Container/Ubuntu20.04/scripts/set_hugepages.sh

```diff
@@ -0,0 +1,16 @@
+#Copyright (C) 2021 Intel Corporation
```

Comment:
Better to reuse the existing utility script. You can modify the Docker file to refer to the existing common file during the docker build procedure...

Reply:
Removed from Ubuntu20.04 folder.